jupyter-docker/run-jupyter-local.sh

```diff
@@ -12,13 +16,14 @@ start () {
     fi
 
     echo "Starting Jupyter server container..."
-    docker create -it --name ${CONTAINER} --rm -p 8888:8888 $DOCKER_IMG 'export PYTHONPATH=$PYTHONPATH:/etc/jupyter/custom;start-notebook.sh'
+    docker create -it --rm --name ${CONTAINER} -p 8000:8000 -e GOOGLE_PROJECT=project -e CLUSTER_NAME=cluster "${DOCKER_IMG}"
```

Comment:
This means the notebook server will listen at `http://localhost:8000/notebooks/project/cluster` :)